automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -112,14 +111,37 @@ object Leonardo extends WorkbenchClient with LazyLogging {
   }
 
   object notebooks {
+    def handleContentItemResponse(response: String): ContentItem = {
+      mapper.readValue(response, classOf[ContentItem])
+    }
+
     def notebooksPath(googleProject: GoogleProject, clusterName: ClusterName): String =
       s"notebooks/${googleProject.value}/${clusterName.string}"
 
+    def contentsPath(googleProject: GoogleProject, clusterName: ClusterName, contentPath: String): String =
+      s"${notebooksPath(googleProject, clusterName)}/api/contents/$contentPath?content=1"
+
+    def localizePath(googleProject: GoogleProject, clusterName: ClusterName): String =
+      s"${notebooksPath(googleProject, clusterName)}/api/localize"
+
     def get(googleProject: GoogleProject, clusterName: ClusterName)(implicit token: AuthToken, webDriver: WebDriver): NotebooksListPage = {
       val path = notebooksPath(googleProject, clusterName)
       logger.info(s"Get notebook: GET /$path")
       new NotebooksListPage(url + path)
     }
 
+    def localize(googleProject: GoogleProject, clusterName: ClusterName, locMap: Map[String, String])(implicit token: AuthToken): String = {
+      val path = localizePath(googleProject, clusterName)
+      logger.info(s"Localize notebook files: POST /$path")
+      val cookie = Cookie(HttpCookiePair("FCtoken", token.value))
+      postRequest(url + path, locMap, httpHeaders = List(cookie))
```

Comment:
this adds both the Auth: Bearer header and the cookie header but... eh, it'll do

Reply:
double auth